src/EditorFeatures/CSharpTest/CodeActions/SyncNamespace/SyncNamespaceTests_ActionCount.cs

```diff
@@ -0,0 +1,293 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
name of this file is weird.  it doesn't actually seem to eb testing 'action count'.